docs/lib/auth/fragments/flutter/signin/30_signIn.md

```diff
@@ -12,4 +12,7 @@ try {
 }
 ```
 
+<amplify-callout>
+Please note that you will be prevented from successfully calling `signIn` if a user has already signed in and a valid session is active.  You must first call `signOut` to remove the original session.  When running on the iOS platform, you will be able to call `signIn` if the session has expired, while on Android you must first call `signOut` regardless. 
```

Comment:
Is there a disparity still? I think you can leave it at call signOut before signIn?